src/components/Dialogs/index.jsx

```diff
@@ -29,38 +28,64 @@ const MessageItem = (props) => {
   return <div className={styles.message}>{text}</div>;
 };
 
-const areaRef = createRef();
+// const areaRef = createRef();
 
-const onClickHandler = () => {
-  const areaValue = areaRef.current?.value;
-  alert(areaValue);
-};
+// const onClickHandler = () => {
+//   const areaValue = areaRef.current?.value;
+//   alert(areaValue);
+// };
+
+// const onChangeHandler = (event) => {
+//   console.log(event.target.value);
+// };
 
-const onChangeHandler = (event) => {
-  console.log(event.target.value);
+let onSendMessageClick = () => {
+  store.dispatch(sendMessageCreator());
+};
+let onNewMessageChange = (event) => {
+  let body = event.target.value;
+  store.dispatch(updateNewMessageBodyCreator(body));
 };
 
 const Dialogs = ({ list }) => {
   return (
     <div className={styles.dialogs}>
       <div className={styles.dialogsItems}>
         <DialogItem
-          name={dialogsData.map((dia, index) => (
-            <li key={index}>{dia.name}</li>
+          name={store._state.DIALOGS.CONVERSATIONS.map((dia, index) => (
```

Comment:
dia != dialog, why it can`t be just the full name of the variable? Just use "dialog" instead of unclear "dia"